src/components/CompareResults/CompareResultsTable.tsx

```diff
@@ -4,49 +4,39 @@ import CircularProgress from '@mui/material/CircularProgress';
 import Grid from '@mui/material/Grid';
 import Paper from '@mui/material/Paper';
 import Table from '@mui/material/Table';
-import TableCell from '@mui/material/TableCell';
 import TableContainer from '@mui/material/TableContainer';
-import TableHead from '@mui/material/TableHead';
-import TableRow from '@mui/material/TableRow';
 
+import { RootState } from '../../common/store';
 import { useAppSelector } from '../../hooks/app';
-import type { CompareResultsState } from '../../types/state';
-import type { CompareResultsTableHeaders } from '../../types/types';
+import type {
+  CompareResultsState,
+} from '../../types/state';
+import CompareResultsTableHead from './CompareResultsTableHead';
+import CompareTableStatus from './CompareTableStatus';
 import PaginatedCompareResults from './PaginatedCompareResults';
 
-const tableHeaders: CompareResultsTableHeaders[] = [
-  { id: 'platform', label: 'Platform', align: 'center' },
-  { id: 'graph', label: 'Graph', align: 'center' },
-  { id: 'test-name', label: 'Test Name', align: 'left' },
-  { id: 'base-value', label: 'Base', align: 'center' },
-  { id: 'new-value', label: 'New', align: 'center' },
-  { id: 'delta-percent', label: 'Delta', align: 'center' },
-  { id: 'confidence', label: 'Confidence', align: 'center' },
-  { id: 'total-runs', label: 'Total Runs', align: 'center' },
-];
-
 function CompareResultsTable(props: CompareResultsProps) {
   const { mode } = props;
   const compareResults: CompareResultsState = useAppSelector(
-    (state) => state.compareResults,
+    (state: RootState) => state.compareResults,
```

Comment:
since we are using typed hooks (`useAppDispatch` and `useAppSelector`) you shouldn't need to define a type here: https://redux-toolkit.js.org/tutorials/typescript#define-typed-hooks

are you getting an error in your editor or lint command?

Reply:
I was getting the errors from the editor (Visual Studio Code). It seems to happen to other people as well. https://stackoverflow.com/questions/51891407/ts-lint-in-visual-studio-code-suddenly-indicates-wrong-lint-errors

Reply:
I've had problems with this at times also. which linter extension are you using?

Reply:
I have the eslint extension and the @builtin extensions for typescript in vscode. The npm run lint doesn't give any linting error.